.bazelrc

```diff
@@ -0,0 +1,9 @@
+# Configurations for arguments that should automatically be added to Bazel commands.
```

Comment:
As per point number 8 in the PR description, are we planning to introduce `.bazelignore` and moving .gitignore out?

```
This PR also introduces updates to .gitignore and a new .bazelrc file to simplify working with 
Bazel. The latter will be iterated on as we approach moving the entire team over to Bazel 
permanently.
```

Reply:
Do we actually need anything in ``.bazelignore`` at the moment? My point was just meaning that we'll add more to both ``.bazelrc`` and ``.gitignore`` as we move over to Bazel permanently--given that everything builds, I don't think anything needs to be ignored by Bazel currently.